app/config.py

```diff
@@ -472,7 +472,7 @@ class Production(Config):
 
 
 class Staging(Production):
-    pass
+    NOTIFY_ENVIRONMENT = "staging"
```

Comment:
If you look at lines above, `NOTIFY_ENVIRONMENT` was set to a constant in production and not using the env variable. 😢